src/libraries/System.Security.Cryptography/tests/SlhDsaOpenSslTests.cs

```diff
@@ -57,7 +57,7 @@ public void SlhDsaOpenSsl_DuplicateKeyHandleLifetime()
 
                 using (one)
                 {
-                    Assert.Equal(oneSignature.Length, one.SignData(data, oneSignature, context));
+                    oneSignature = one.SignData(data, context);
```

Comment:
Line 56 allocates a new array... and then we immediately overwrite the local. Should we get rid of the `new byte[...]` on line 56?